nxviz/plots.py

```diff
@@ -338,10 +340,18 @@ def __init__(self, graph, **kwargs):
         Accepts the following additional arguments apart from the ones in
         `BasePlot`:
 
-        :param rotate_labels: Whether to rotate node labels.
-        :type node_color: `bool`
+        :param node_layout: which/whether (a) node layout is used,
+            either 'rotation', 'numbers' or None
+        :type node_layout: `string`
         """
-        self.rotate = kwargs.pop("rotate_labels", False)
+
+        # Store node layout
```

Comment:
If possible, could you elaborate upon the comment line for documentation purposes? I'm preparing for the possibility of forgetting what this block of code was all about 😅. Here's my attempt as an example:

```
# Specify what the node label layout will be. 
# Only three are provided: None, 'rotation', and 'numbers'. 
# Rotation: labels are drawn along the radial axis with appropriate rotation and alignment (e.g. a label at the 2 o'clock position will be rotated +30º from the horizontal and left-aligned, while a label at the 10 o'clock position will be rotated -30º from the horizontal and right-aligned).
# Numbers: labels are rendered along the circumference with appropriate alignment, but are not rotated (i.e. they are either horizontal left-aligned or horizontal right-aligned).
```